product_attribute_global_item_code/product_view.xml

```diff
@@ -0,0 +1,28 @@
+<openerp>
+    <data>
+        <record id="product_template_only_form_view" model="ir.ui.view">
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="product.product_template_only_form_view"/>
+            <field name="arch" type="xml">
+            <xpath expr="//p" position="after">
```

Comment:
I think searching for all paragraph tags and relying on their only being one is a bit dangerous.  How about better qualifying it e.g. //page[@name='variants']/p or even putting using before and using the attribute_line_ids for the parse.